src/components/BoardHeader/BoardHeader.scss

```diff
@@ -90,15 +85,18 @@
     transform: scale(1);
   }
 }
+.board-header__share-button > svg {
+  height: 36px;
+  width: 36px;
+}
 
 [theme="dark"] {
   .board-header__share-button {
-    color: $color-white;
-    border: 2px solid $color-planning-pink;
+    border: 2px solid #9ebfff;
+    color: #9ebfff;
```

Comment:
I'd prefer to add this color to the `style.scss` and use variables here.

Reply:
for the readiness circle, i named the color: `color-icon-light-blue`

Reply:
I've added it to the style constants 👍🏼